api/v1beta1/ovncontroller_types.go

```diff
@@ -75,6 +75,11 @@ type OVNControllerSpecCore struct {
 	// +optional
 	NicMappings map[string]string `json:"nicMappings,omitempty"`
 
+	// +kubebuilder:validation:Optional
+	// +optional
+	// Map of bond names to their configuration (links and mode)
```

Comment:
can remove links and mode from the description, else we have to add mtu or any other future bond option here.